conda-forge.yml

```
@@ -9,3 +9,7 @@ os_version:
   linux_64: cos7
 conda_build:
   pkg_format: '2'
+build_platform:
+  osx_arm64: osx_64
+  linux_aarch64: linux_aarch64
```

```suggestion
  linux_aarch64: linux_64
```

You are now building linux aarch64 "natively", well really, using "emulation + qemu".

You can go back to cross compilation with this change + rerender

---

you can also build locally with


```
python build-locally.py
```

or:

```
conda debug -m .ci_support/THECONFIG.yaml recipe
```